.gitignore

```diff
@@ -10,3 +10,10 @@
 /sourcemap.json
 
 /Packages
+/DevPackages
+
+
+/test/Packages
```

Comment:
This can be deleted

Reply:
/test/Packages yes, but DevPackages we need, since dev-dependencies goes into devpackages

Reply:
Yes, I was talking only about /test/Packages